crowbar_framework/app/controllers/backups_controller.rb

```diff
@@ -63,18 +63,21 @@ def create
   # /utils/backup/restore   POST   Trigger a restore
   def restore
     respond_to do |format|
-      if @backup.restore(background: false)
-        format.json { head :ok }
-        format.html { redirect_to backups_path }
-      else
-        format.json do
-          render json: { error: @backup.errors.full_messages.first }, status: :unprocessable_entity
-        end
-        format.html do
+      format.html do
+        if @backup.restore(background: false)
+          redirect_to backups_path
```

Comment:
Style/IdenticalConditionalBranches: Move `redirect_to backups_path` out of the conditional.